multi_import/cache.py

```diff
@@ -1,8 +1,7 @@
 from collections import defaultdict
 
-from django.core.exceptions import MultipleObjectsReturned
 import six
-from tablib.compat import unicode
```

Comment:
Replaced unicode with str similar to the tablib repo - https://github.com/jazzband/tablib/pull/408.